linalg/amgxsolver.cpp

```diff
@@ -30,10 +30,15 @@ int AmgXSolver::count = 0;
 
 AMGX_resources_handle AmgXSolver::rsrc = nullptr;
 
+AmgXSolver::AmgXSolver()
+   : ConvergenceCheck(false) {};
+
 AmgXSolver::AmgXSolver(const AMGX_MODE amgxMode_, const bool verbose)
```

Comment:
I think you need to initialize `ConvergenceCheck` to `false` in every constructor (or have an `else` in the `if amgxMode` line).

Reply:
Ah that is right, fixed!